src/ds/ds6/ds6-private.h

```diff
@@ -94,8 +120,7 @@ namespace librealsense
             {
                 for (auto i = 0; i < max_ds_rect_resolutions; i++)
                     depth_intrinsic[i] = {};
-                data_present.emplace(ds6_calibration_table_id::coefficients_table_id, false);
-                //data_present.emplace(ds6_calibration_table_id::depth_calibration_id, false);
+                data_present.emplace(ds6_calibration_table_id::depth_calibration_id, false);
                 data_present.emplace(ds6_calibration_table_id::rgb_calibration_id, false);
                 //data_present.emplace(ds6_calibration_table_id::fisheye_calibration_id, false);
```

Comment:
Do we need lines 124 + 125?
Are we missing something currently?

Reply:
Honestly, no one is using this ds6_calibration class. Same for ds5_calibration class.
We need to understand where these come from.
But this is not related to this PR, I believe.